test/js_optimizer/test-object-literals.js

```diff
@@ -0,0 +1,14 @@
+var foo = 1;
+var baz = {};
+
+// Verify that shorthand object literals are preserved.
+var bar = {
+  foo,
+  baz,
+};
+
+// Objects that could use shorthand are also converted where possible.
+var bar2 = {
+  foo: foo,
+  baz: baz,
```

Comment:
Maybe add one that can't be converted?
```suggestion
  baz: baz,
  other1: other2,
```